extension-manifest-v3/test/store/pre-setup.js

```diff
@@ -0,0 +1,6 @@
+import chrome from 'sinon-chrome';
+
+window.document = {};
```

Comment:
required for `hybrids` to prevent:
```
./test/store/options.test.js (uncaught error)
error: TypeError: Cannot read properties of undefined (reading 'adoptedStyleSheets')
    at file:///Users/chrmod/Projects/github.com/ghostery/ghostery-extension/extension-manifest-v3/node_modules/.deno/hybrids@8.1.15/node_modules/hybrids/src/template/layout.js:3:49
This error was not caught from a test and caused the test runner to fail on the referenced module.
It most likely originated from a dangling promise, event/timeout handler or top-level code.
```